tests/defined_resource_types_spec.rb

```diff
@@ -1,13 +1,13 @@
 describe "Task 1:" do
-  it 'Create the directory structure for your accounts module' do
+  it 'Create the directory structure for your accounts module' host: :localhost do
```

Comment:
Do it work to put the `host:` clause on the `it` line, or should it go with `describe`? I've been putting mine on the `describe` line, but maybe this syntax is OK, too.